guides/customGuideAssets/ImprovedProgressBar/readme.md

```diff
@@ -0,0 +1,10 @@
+# Custom Progress Bar
```

Comment:
This snippet is super cool. No changes needed here.

Reply:
@fortynachos awesome.